google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -79,7 +79,23 @@
         `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
         `group_id` varchar(255) DEFAULT NULL,
         `group_email` varchar(255) DEFAULT NULL,
+        `group_kind` varchar(255) DEFAULT NULL,
+        `direct_member_count` bigint(20) DEFAULT NULL,
         `raw_group` json DEFAULT NULL,
         PRIMARY KEY (`id`)
     ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 """
+
+CREATE_GROUP_MEMBERS_TABLE = """
+    CREATE TABLE `{0}` (
+        `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+        `group_id` varchar(255) DEFAULT NULL,
+        `member_kind` varchar(255) DEFAULT NULL,
+        `member_role` varchar(255) DEFAULT NULL,
+        `member_type` varchar(255) DEFAULT NULL,
+        `member_status` varchar(255) DEFAULT NULL,
+        `member_id` varchar(255) DEFAULT NULL,
+        `raw_member` json DEFAULT NULL,
```

Comment:
In addition to this here, we also should have the entirety of the unparsed (i.e. raw) Member response stored in one big chunk in a separate table, with a 1:1 relation to Groups.  Please take a look at the RAW_PROJECT_IAM_POLICIES_TABLE.  Okay, to stick a TODO here and follow-up.

This is by design, as we might need it in case of troubleshooting our pipeline.

Storing this in a separate table is non-ideal, but it's a trade-off between the speed of loading to separate table, vs inserting into the Groups table.